Jenkinsfile_promote_salt_packages

```diff
@@ -58,6 +58,7 @@ pipeline {
                 echo 'Run services for SUSE-Manager-Tools Salt packages'
                 sh "osc -A https://api.suse.de service remoterun Devel:Galaxy:Manager:${params.manager_tools_source_version}:Ubuntu16.04-SUSE-Manager-Tools salt"
                 sh "osc -A https://api.suse.de service remoterun Devel:Galaxy:Manager:${params.manager_tools_source_version}:Ubuntu18.04-SUSE-Manager-Tools salt"
+                sh "osc -A https://api.suse.de service remoterun Devel:Galaxy:Manager:${params.manager_tools_source_version}:Ubuntu18.04-SUSE-Manager-Tools salt"
```

Comment:
This should be 20.04 and not 18.04, right?

Reply:
Argh! Yeah, it's a typo.

Reply:
Fixed.